app/views/admins/user_search.mobile.haml

```diff
@@ -0,0 +1,56 @@
+.col-md-9
+  .row
+    .user_search.col-md-8
+      %h3= t('admins.admin_bar.user_search')
+      = form_for @search, url: {action: 'user_search'}, html: {method: :get, class: 'form-horizontal'} do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.